consensus/hotstuff/cruisectl/config.go

```diff
@@ -1,27 +1,23 @@
 package cruisectl
 
 import (
-	"math"
 	"time"
 )
 
 // DefaultConfig returns the default config for the BlockRateController.
 func DefaultConfig() *Config {
 	return &Config{
-		TargetTransition: EpochTransitionTime{
-			day:    time.Wednesday,
-			hour:   19,
-			minute: 0,
-		},
+		TargetTransition: DefaultEpochTransitionTime(),
 		// TODO confirm default values
 		DefaultProposalDelay: 500 * time.Millisecond,
 		MaxProposalDelay:     1000 * time.Millisecond,
 		MinProposalDelay:     250 * time.Millisecond,
```

Comment:
This represents the total time from entering view V to broadcasting a proposal for view V, it is not a delay added on top. Therefore, practically speaking, 250ms is the same as 0ms.

More importantly, to me these bounds represent safety threshold against unexpected behaviour. I think the defaults should be restrictive and we can configure it to be less restrictive, at least at first, until we have some real-world experience of the controller's behaviour. 

In practice, if the controller is functioning as expected and the network is not under extreme and unusual stress, it should require very little freedom to maintain target switchover times.

Reply:
I think your argument is very applicable to this situation: 
> the simulation and spec dictate that the "default" `ProposalDuration` value (the time between entering a view and broadcasting a proposal for that view) should be τ_0, the "ideal view time". However, the duration which we are controlling is significantly less than our computed τ_0.
> * τ_0 is the average time for a complete view, from entering to exiting
> * `ProposalDuration` - the value we control - is only a fraction of a view time (in the ballpark of 50%)

Exactly for this reason you are outlining, 50% of the view corresponding to 250ms, i.e. 100% of the view corresponding to 500ms. In practise, the controller will rarely be able to go faster on mainnet. So I don't think it would make a difference for mainnet. 

_However_, when we test the controller on Canary, it _can_ probably go faster. I'd rather allow the controller to go faster on canary and test that it behaves safely. In comparison, _not_ testing it on canary would create more uncertainty. If we ever had the case on mainnet that we needed to go faster, we wouldn't know whether we safely could reduce `DefaultProposalDelay`

Reply:
While I would prefer to use lower values, I also follow your argument.

Reply:
>  I'd rather allow the controller to go faster on canary and test that it behaves safely

We can do this by adjusting the configs, but I don't think we should change the _defaults_ in order to facilitate Canary testing.